REFERENCE.md

```diff
@@ -0,0 +1,195 @@
+# Learnosity Node.js-SDK: Reference guide
+
+## Examples
+You can find a complete site with examples of Learnosity integration in our [demo site](http://demos.learnosity.com/).
+
+You can download the entire site or browse the code directly on [GitHub](https://github.com/Learnosity/learnosity-demos/).
+
+## Usage
+
+### Learnosity.init()
+
+The init function is used to create the necessary *security* and *request* details used to integrate with a Learnosity API. Most often this will be a JavaScript object.
+
+The init function takes up to 5 arguments:
+
+ * [string]  service type
+ * [array]   security details (**no secret**)
+ * [string]  secret
+ * [request] request details *(optional)*
+ * [string]  action *(optional)*
+
+```
+Structure of Node.js project (based on Express.js and EJS template):
+- node_modules
+----- ejs
+----- express
+----- learnosity-sdk-nodejs
+----- (all standard modules)
+- views
+----- index.ejs
+- package.json
+- package-lock.json
+- app.js
+```
+
+``` javascript
+app.js:
+
+var Learnosity = require('learnosity-sdk-nodejs');
+var express = require('express');
+var app = express();
+
+app.set('view engine', 'ejs');
+
+app.get('/', function (req, res) {
+    var learnositySdk = new Learnosity();
+    var request = learnositySdk.init(
+        // service type
+        "questions",
+
+        // security details
+        {
+            "consumer_key": "yis0TYCu7U9V4o7M",
+            "domain":       "localhost",
+            "user_id":      "$ANONYMIZED_USER_ID"
```

Comment:
It would probably be better to use the quick-and-dirty UUID generator `const uuid = require('uuid')`, as in the standalone-assessment example.

Reply:
I fixed it.